### PR TITLE
Add new `Minitest/AssertPathExists` and `Minitest/RefutePathExists` cops

### PR DESCRIPTION
**Pull request by fatkodima**
https://github.com/rubocop-hq/minitest-style-guide#assert-path-exists

Both cops are almost identical (same as `AssertNil` and `RefuteNil`, for example). Is it worth extracting (and complicating a bit) somehow into, say, module?

### CODE REVIEW

**Comment by fatkodima** [comment]
Rebased.

File: config/default.yml [hunk 1]
```diff
@@ -51,6 +51,12 @@ Minitest/AssertNil:
   Enabled: true
   VersionAdded: '0.1'
 
+Minitest/AssertPathExists:
+  Description: 'This cop enforces the test to use `assert_path_exists` instead of using `assert(File.exist?(path))`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-path-exists'
```
Comment (bbatsov) [inline config/default.yml:56]: It's generally better to link to the website instead - https://minitest.rubystyle.guide/ The guide looks much nicer there.

File: config/default.yml [hunk 2]
```diff
@@ -117,6 +123,12 @@ Minitest/RefuteNil:
   Enabled: true
   VersionAdded: '0.2'
 
+Minitest/RefutePathExists:
+  Description: 'This cop enforces the test to use `refute_path_exists` instead of using `refute(File.exist?(path))`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#refute-path-exists'
```
Comment (bbatsov) [inline config/default.yml:128]: Same here.

**Comment by fatkodima** [comment]
Updated links.

**Comment by koic** [comment]
> Is it worth extracting (and complicating a bit) somehow into, say, module?

This is an issue with Minitest RuboCop, but this PR is fine as a first step.

**Comment by koic** [comment]
Thank you!

**Pull request merged**